packages/contact-exchange/src/services/HandleOOBInvitation.ts

```diff
@@ -0,0 +1,55 @@
+import { Wallet } from './Wallet';
+import { OutOfBandInvitation, OutOfBandService } from './DIDCommOOBInvitation';
+
+export function handleOOBInvitation(
+  wallet: Wallet,
+  invitation: OutOfBandInvitation | string,
+  identity: string,
+): void {
+  try {
+    let parsedInvitation: OutOfBandInvitation | null = null;
+
+    if (typeof invitation === 'string' && invitation.startsWith('{')) {
+      try {
+        parsedInvitation = JSON.parse(invitation);
+      } catch (error) {
+        console.error('Error parsing invitation:', error);
+        return;
+      }
+    } else if (typeof invitation === 'object') {
+      parsedInvitation = invitation;
+    } else {
+      console.error('Invalid invitation type:', typeof invitation);
+      return;
+    }
+
+    // Validate the parsed invitation
+    if (!parsedInvitation?.services || parsedInvitation.services.length === 0) {
```

Comment:
The services field is not a mandatory field on the spec of an OOB:

https://identity.foundation/didcomm-messaging/spec/#invitation

This is how a valid OOB looks like:

![image](https://github.com/user-attachments/assets/ead20ad4-0a23-4ecc-b951-57c235c12e5e)

Please tests your whole flow and the processOOBInvitation() method with one valid OOB invitation.